src/jax.jl

```diff
@@ -26,17 +24,17 @@ end
 function (wrap::JaxFunctionWrapper)(args...)
     # TODO: handle multiple outputs
     out = (wrap.jaxfn(mapover(x->jax.numpy.asarray(PyReverseDims(x)), x-> x isa Array, args)...))
-    return ReverseDimsArray(via_dlpack(dlpack, out))
+    return DLArray(out, pyto_dlpack).data
```

Comment:
`out` can be garbage collected if you don't keep it around so you might see memory corruption as you are doing. A way to do this is to return the whole `DLArray` and get its `data` field outside the function call.

Reply:
Another alternative is to manually increase the counter of the python object, but then you need to find a way to do eventually decrease it when you no longer need it, which is harder than keeping the `DLArray` around.